src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec

```diff
@@ -12,7 +12,7 @@
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4. </summary>
     <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4. Targets .NET 4.5 and .NET Platform Standard 1.1.
-OData .NET library is open source at http://github.com/OData/odata.net. Documentation for the library can be found at https://odata.github.io/odata.net.</description>
+OData .NET library is open source at http://github.com/OData/odata.net. Documentation for the library can be found at https://docs.microsoft.com/en-us/odata/.</description>

```

Comment:
>https://docs.microsoft.com/en-us/odata/ [](start = 118, length = 39)

Should we point this more specifically to documentation for ODataClient: [https://docs.microsoft.com/en-us/odata/client/getting-started](https://docs.microsoft.com/en-us/odata/client/getting-started)?